app/shared/ui-settings/src/commonMain/kotlin/ui/settings/tabs/theme/ThemePreferences.kt

```diff
@@ -165,7 +162,7 @@ private fun ColorButton(
     color: Color, themeSettings: ThemeSettings, state: SettingsState<ThemeSettings>
 ) {
     ColorButton(
-        modifier = Modifier,
+        modifier = Modifier.padding(4.dp),
```

Comment:
No 'hidden' paddings (unless you are comply to minimum interaction size requirement (48.dp)). Let the caller decide (add a `modifier` param).